playbooks/common/openshift-master/post_config.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Configure Node Labels
+  hosts: oo_masters_to_config
+  roles:
+    - openshift_label_nodes
```

Comment:
This should probably be added to the common/openshift-node/config.yml playbook, we are already switching between host types in there to handle the node registration, might as well do the node labeling there as well.

Also, do we want to support modifying node labels after node creation?  If so, then having the openshift_label_nodes role makes sense, if not, then we should just use the openshift_node_labels and do it at node registration.